### ISSUE TITLE
Add aria-current to the active link in the selectable list component

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#172]
### Problem

[molecules/molecules.selectable-list](https://a-ui.github.io/core_branding_scss/#molecules-selectable-list)

1. The active link is only indicated visually (`class="m-selectable-list__item--active"`). A screenreader user cannot know it's the active link.
2. The visual state of the active link scores below the required ratio of 3:1 for non-text content. `<li>` border color #949494 against `<li>` background color #F3F3F3 scores 2,7.

![screenshot list](https://user-images.githubusercontent.com/870049/99261841-86766200-281d-11eb-901a-91c243aaaa07.jpg)

![contrast](https://user-images.githubusercontent.com/870049/99261761-72cafb80-281d-11eb-89cf-9c65044bee34.jpg)

### Solution

1. Add aria-current="page" to the active link.
2. Enhance the contrast between li background and border or add an another affordance that doesn't rely on color. For example: an icon, bold text or something else.

### Links and articles

[WAI-ARIA 1.2: aria-current (state)](https://www.w3.org/TR/wai-aria-1.2/#aria-current)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)

**Comment by Veyfeyken** [comment]
Aria-current is present but the colours are still the same? Create a separate issue for design?

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken In your solution you gave the option to add bold as a valid alternative of enhancing the contrast, which I used to distinguish the active link. Isn't this enough? Can you give guidance on what else should be done? Thanks in advance!

**Comment by Veyfeyken** [comment]
I'm sorry, I completely missed that. I was too focused on the background and border colors. The bold text does the trick.
